x/staking/keeper/querier.go

```diff
@@ -132,6 +136,9 @@ func queryValidatorUnbondingDelegations(ctx sdk.Context, req abci.RequestQuery,
 	}
 
 	unbonds := k.GetUnbondingDelegationsFromValidator(ctx, params.ValidatorAddr)
+	if unbonds == nil {
+		unbonds = make(types.UnbondingDelegations, 0)
```

Comment:
ditto (and so on) - the point is that across the codebase we generally don't initialise types with `make()`

Reply:
Should've fixed all of them. Thanks for the tip!